tests/unit-tests/test_config.py

```diff
@@ -1241,6 +1242,8 @@ def create_gnn_config(tmp_path, file_name):
     }
     yaml_object["gsf"]["gnn"] = {
         "node_feat_name": ["ntype0:feat_name"],
+        "edge_feat_name": ["ntype0, rel0, ntype1:feat_name"],
```

Comment:
Do we allow spaces?

Reply:
yes. This implementation will trim white spaces after splitting.